tests/Unit/LocaleTest.php

```diff
@@ -31,20 +31,26 @@ public function testEnglishHungarianConsistency()
             if($file_in == 'validation.php') {
                 $this->assertEquals(array_keys($en_expressions['attributes']), array_keys($hu_expressions['attributes']));
             }
-            // The key names do not contain any points.
-            foreach ($en_expressions as $key => $value) {
-                $this->assertFalse(strpos($key, '.'));
-                if($file_in == 'validation.php') {
-                    foreach ($en_expressions['attributes'] as $key2 => $value2) {
-                        $this->assertFalse(strpos($key2, '.'));
-                    }
-                }
-            }
-            foreach ($hu_expressions as $key => $value) {
-                $this->assertFalse(strpos($key, '.'));
-                if($file_in == 'validation.php') {
-                    foreach ($en_expressions['attributes'] as $key2 => $value2) {
-                        $this->assertFalse(strpos($key2, '.'));
+        }
+    }
+
+    /**
+     * This test is to ensure that hungarian and english locale keys do not contain any point.
+     *
+     * @return void
+     */
+    public function testLocaleKeyFormats()
+    {
+        foreach (['hu', 'en'] as $lang){
```

Comment:
Thinking more about it, it would be enough to check only one of these, since above we assert that they are the same. But we do not lose anything with running for both and might save some time if we remove the above assertion .